README.md

```diff
@@ -97,7 +97,7 @@ The app signature will be broken if the .apk is altered in any way — unsigned
 
 ```kotlin
 piracyChecker {
-	enableSigningCertificate("478yYkKAQF+KST8y4ATKvHkYibo=") // The original APK signature for the PRODUCTION version
+	enableSigningCertificates("478yYkKAQF+KST8y4ATKvHkYibo=") // The original APK signature for the PRODUCTION version
```

Comment:
Could you add an example with multiple signatures?

Reply:
I can't sorry... I don't really have experience with multiple signatures. Just implemented the new methods following documentation 😬

Reply:
No problem. It will be useful to allow different variants (such as debug and release signatures).